### PR TITLE
Test syntax

### PR DESCRIPTION
**Pull request by vnugent**

### CODE REVIEW

**Comment by gibboj** [comment]
Sorry I updated your branch. I moved to my own branch, just pulled in the develop branch (poorly somehow...)

File: .github/workflows/auto-version-bump.yml [hunk 1]
```diff
@@ -30,7 +32,7 @@ jobs:
 
       # Setup .npmrc file to publish to npm
       - name: Setup Node.js environment
-        if: $NEW_TAG != $OLD_TAG
+        if: steps.version-bump.outputs.newTag != steps.version-bump.outputs.current
```
Comment (vnugent) [inline .github/workflows/auto-version-bump.yml]: this won't work because `steps.version-bump.outputs.newTag` the action doesn't expose it :(

Comment (gibboj) [reply .github/workflows/auto-version-bump.yml]: it "works", but maybe not in a way we want. It does bump the version, but it doesn't commit it back to the package file. So you have to run that commit *after* it publishes.

Comment (gibboj) [reply .github/workflows/auto-version-bump.yml]: Anyway 0.0.8 is published and it builds but there is a vercel error, not sure what it is: https://github.com/OpenBeta/open-tacos/pull/204

**Pull request merged**